src/components/LongTermAverageGraph/LongTermAverageGraph.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {entries, keys} from 'lodash';
+
+
+//this weird sequence avoids crashing npm when plotly is loaded.
+import Plotly from 'plotly.js';
+import createPlotlyComponent from 'react-plotly.js/factory';
+const Plot = createPlotlyComponent(Plotly);
+
+
+function LongTermAverageGraph({longTermData}) {
+
+    /* data is a temporary variable that represents the data, which are
+     key value pairs. The keys are the date of the measurement, the values
+     are the temperature at that date. The keys will be sorted, and then the
+     temperatures will be matched to the correct years. */
+    
+    var dataArray = entries(longTermData);
+    var data = [];
+    var graphTitle = "Mean Long Term Maximum Temperature: ";
+    
+    // Assert that longTermData is formatted as expected and extract important info
+    if (dataArray.length >= 1) {
+        if (dataArray[0].length == 2){
+            data = dataArray[0][1].data;
+            graphTitle = graphTitle.concat(dataArray[0][0].toUpperCase());
```

Comment:
One small change here: please use a javascript template literal instead, they're easier for other people to read.